aten/src/ATen/native/native_functions.yaml

```diff
@@ -1006,6 +1006,10 @@
 - func: layer_norm(Tensor input, int[] normalized_shape, Tensor? weight=None, Tensor? bias=None, float eps=1e-05, bool cudnn_enable=True) -> Tensor
 
 - func: linear(Tensor input, Tensor weight, Tensor? bias=None) -> Tensor
+  dispatch:
```

Comment:
@gchanan do you know why we generated it as `torch.linear` and not `torch._C.nn.linear`?

Reply:
the latter seems more reasonable, similar to `torch._C.nn.nll_loss`

Reply:
all we need to do is add a `python_module: nn`, and I think we should do that in PR as well Junjie:
https://github.com/pytorch/pytorch/blob/f1cb2c3b3d9ce92d018fe55ab907248d4554a07d/aten/src/ATen/native/native_functions.yaml#L3431-L3432

Reply:
@soumith sure just added

Reply:
I think we added `linear` before we had support for `python_module: nn`.